drivers/espi/Kconfig.it8xxx2

```diff
@@ -21,6 +21,81 @@ config ESPI_PERIPHERAL_HOST_IO
 config ESPI_PERIPHERAL_DEBUG_PORT_80
 	default y
 
+config ESPI_PERIPHERAL_EC_HOST_CMD
+	default y
+
+choice
```

Comment:
Add a  label to the choice.  This allows boards to override if needed.
```
choice IT8XXX2_H2RAM_HC_SIZE_CHOICE
```

Reply:
Done.